tests/manage/mcg/conftest.py

```diff
@@ -104,6 +108,176 @@ def bucket_cleanup():
     return _create_buckets
 
 
+@pytest.fixture()
```

Comment:
the current fixture scope is 'function' (the default). Is this the intention?

Reply:
this is the way i saw the other factory was implemented in mcg, should it be different?